src/components/Tooltip/Tooltip.tsx

```diff
@@ -1,13 +1,13 @@
 import React, { FC, useEffect, useRef, useState } from 'react';
 import {
-    useFloating,
-    shift,
-    autoUpdate,
     arrow,
+    autoUpdate,
     offset as fOffset,
+    shift,
+    useFloating,
```

Comment:
If we want to all get on the same page with alphasorting, etc, there is a vscode plugin called TypeScript Hero which will auto sort import statements.

I believe you were opposed to adding a .vscode configuration file that would recommend extensions? Lets revisit that conversation soon. Its a relatively common thing for open source projects and can make onboarding new contributors easier.

Reply:
sure we can discuss that, but i dont use vscode 😄